docs/examples/Thrusties/src/Thrusties.ls

```diff
@@ -55,6 +57,11 @@ package
 			demo.run();
 		}
 		
+        [Test]
+        function thrusty() {
```

Comment:
Maybe comment that this is a test function to make it easier to test loom test command? And/or can it be in the test example?